### PR TITLE
solved Linking a case to an event breaks when there is no event in the db issue #2684

### PR DESCRIPTION
**Pull request by john1496**
… to create event #2684

### CODE REVIEW

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/eventLink/EventListComponent.java [hunk 1]
```diff
@@ -37,14 +43,29 @@
...
 	private EventList list;
 	private Button createButton;
 
 	public EventListComponent(CaseReferenceDto caseRef) {
 
-		createEventListComponent(new EventList(caseRef), e -> ControllerProvider.getEventController().selectOrCreateEvent(caseRef));
+		createEventListComponent(new EventList(caseRef), e -> {
+
+			//check if there are active events in the database
+			List<String> events = FacadeProvider.getEventFacade().getAllActiveUuids();
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/caze/eventLink/EventListComponent.java:58]: Please use a "count" method instead for performance reasons.

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Please have a look at the guide to set up your local dev environment again. There's a section on code formatting there. Please configure code formatting in your IDE as specified in the guide and reformat all files that you've changed.

Also, I don't think that displaying a message when there's no event in the database is necessary. This will only happen once on the server (per user) anyway and the information isn't really relevant for the user in the first place.

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/eventLink/EventListComponent.java (hunk 1 above)
Comment (john1496) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/caze/eventLink/EventListComponent.java:58]: ok will do

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Please use code formatting as described in the development environment guide.

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**